libafl/src/state/mod.rs

```diff
@@ -404,6 +439,55 @@ impl<I, C, R, SC> HasExecutions for StdState<I, C, R, SC> {
     }
 }
 
+impl<I, C, R, SC> HasAFLStats for StdState<I, C, R, SC> {
```

Comment:
Should be `HasAflStats` for consistency.
That being said, maybe there is an even nicer name? We'll need that guy everywhere.